training/src/main/java/wolox/training/exceptions/OldPasswordMistatchException.java

```diff
@@ -0,0 +1,20 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.CONFLICT, reason = "Old Password Mismatch")
+public class OldPasswordMistatchException extends Exception {
```

Comment:
Typo in the word "mistatch", should be "mismatch" 😅 Also change it in the imports and in the lines where you throw the exception. Tip: use Ctrl+Shift+F in IntelliJ to search and replace in multiple files of the project.